### PR TITLE
Install npm-merge-driver

### PR DESCRIPTION
**Pull request by spectranaut**
Installed npm-merge-driver as discussed in https://github.com/w3c/aria-practices/pull/1273, a Greenkeeper PR that had a merge conflict. With this utility installed, Greenkeeper merge conflicts will be automatically resolved.

Added documentation to the [infrastructure wiki page](https://github.com/w3c/aria-practices/wiki/APG-Infrastructure#npm-merge-driver).

### CODE REVIEW

**Comment by zcorpan** [comment]
I got an error when trying to rebase or commit in this branch from `.git/hooks/pre-commit` which was trying to run `node_modules/run-node/run-node`, which didn't exist. I added `run-node` to package.json, which made it work.

This branch is now rebased on latest master, with that addition.

**Comment by zcorpan** [comment]
I have also updated the [infrastructure wiki page](https://github.com/w3c/aria-practices/wiki/APG-Infrastructure#run-node) for `run-node`.

**Comment by zcorpan** [comment]
@mcking65, @spectranaut has reviewed my change. Waiting on CI to complete. If it's green, it should be OK to merge.

File: package.json [hunk 1]
```diff
@@ -60,5 +61,8 @@
     "files": [
       "test/tests/*"
     ]
+  },
+  "dependencies": {
+    "npm-merge-driver": "^2.3.5"
```
Comment (nschonni) [inline package.json]: Should this be under "devDependencies" too?

Comment (zcorpan) [reply package.json]: Maybe it should 🙂 Can you file an issue about this, and cc myself and @spectranaut ?

**Pull request merged**